main.go

```diff
@@ -67,14 +70,30 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+func runPrometheusServer(metricsAddr string) {
+	setupLog.Info("Entered runPrometheusServer")
+	mux := restful.NewContainer()
+	mux.Handle("/metrics", promhttp.Handler())
+	setupLog.Info("Defining prom server")
+	server := http.Server{
+		Addr:    metricsAddr,
+		Handler: mux,
+	}
+	setupLog.Info("Starting prom server with TLS")
+	err := server.ListenAndServeTLS(path.Join(sdkTLSDir, sdkTLSCrt), path.Join(sdkTLSDir, sdkTLSKey))
+	if err != nil {
+		setupLog.Error(err, "Failed to start metrics server")
```

Comment:
Or call it metrics everywhere, like here

Reply:
fixed